docs/src/main/asciidoc/security-oidc-bearer-token-authentication.adoc

```diff
@@ -355,9 +355,11 @@ Consider using the `quarkus.oidc.token.audience` property to verify the token `a
 [[single-page-applications]]
 === Single-page applications
 
+==== Use OIDC provider scripts
+
 A single-page application (SPA) typically uses `XMLHttpRequest`(XHR) and the JavaScript utility code provided by the OIDC provider to acquire a bearer token to access Quarkus `service` applications.
 
-For example, if you work with Keycloak, you can use `keycloak.js` to authenticate users and refresh the expired tokens from the SPA:
+For example, if you work with Keycloak, you can use `keycloak.js` to authenticate users using the authorization code flow and refresh the expired tokens from the SPA:
```

Comment:
This seems like a wrong thing to suggest here in this context (Vale again)

Reply:
@FroMage This one looks fine, I've added the clarification how the users are actually authenticated. I suppose `an authorization code flow` instead of `the ...` would be correct though